src/widgetastic_patternfly4/alert.py

```diff
@@ -48,6 +52,14 @@ def click_action(self):
         el = self.browser.element(self.ACTION)
         self.browser.click(el)
 
+    def click_link(self):
+        el = self.browser.element("./span/a", parent=self._raw_description_el)
+
+        if el.is_displayed:
+            self.browser.click(el)
+        else:
+            raise ValueError("Could not find a link")
```

Comment:
What's the point raise `ValueError` here? If `el` is not displayed then `NoSuchElementException` will be raised.
```python
def click_link(self):
    el = self.browser.element("./span/a", parent=self._raw_description_el)
    self.browser.click(el)
```

Reply:
Yes, that is correct
changed as suggested